source/common/common/version.cc

```diff
@@ -17,12 +17,12 @@ const std::string& VersionInfo::revisionStatus() {
 }
 
 std::string VersionInfo::version() {
-  return fmt::format("{}/{}/{}/{}", revision(), BUILD_VERSION_NUMBER, revisionStatus(),
+  return fmt::format("{}/{}/{}/{}/{}", revision(), BUILD_VERSION_NUMBER, revisionStatus(),
```

Comment:
I think someone asked about this recently in Slack, but if the docs for `--version` don't already explain what each field means, can you add the docs and also doc the new SSL component of the version string?